From dae857a94d66839d1623e017e890345f2d81c06e Mon Sep 17 00:00:00 2001
From: Dominique Leuenberger <dimstar@opensuse.org>
Date: Thu, 29 Oct 2015 13:56:18 +0100
Subject: placessidebar: do not crash if uris is NULL

On Drag'n'Drop actions across system boundaries (VM host to guest), the
happen to be null.

https://bugzilla.gnome.org/show_bug.cgi?id=757298
---
 gtk/gtkplacessidebar.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: gtk+3.0-3.16.7/gtk/gtkplacessidebar.c
===================================================================
--- gtk+3.0-3.16.7.orig/gtk/gtkplacessidebar.c
+++ gtk+3.0-3.16.7/gtk/gtkplacessidebar.c
@@ -1883,7 +1883,7 @@ build_file_list_from_uris (const gchar *
   gint i;
 
   result = NULL;
-  for (i = 0; uris[i]; i++)
+  for (i = 0; uris && uris[i]; i++)
     {
       GFile *file;
 
